fmriprep/utils/sentry.py

```diff
@@ -180,3 +180,27 @@ def _chunks(string, length=CHUNK_SIZE):
     """
     return (string[i:i + length]
             for i in range(0, len(string), length))
+
+
+def ping_migas(status='pending'):
```

Comment:
I think we need a few tests here. Can we simulate what happens with `--notrack` set and with no network access?